package.json

```diff
@@ -95,7 +95,7 @@
   "version": "0.0.0",
   "resolutions": {
     "@types/react": "17.0.2",
-    "vite": "4.4.9",
+    "vite": "5.3.1",
     "@oclif/core": "3.26.5",
     "whatwg-url": "14.0.0",
```

Comment:
Could you check if we can remove this? A punycode warning was shown with Node 22, and I think vite was one of the culprits.

Reply:
I don't think it can be removed. The extensions console uses vite to build for instance

Reply:
I meant `whatwg-url`